src-web/containers/AcmGrcPage.js

```diff
@@ -1,25 +1,70 @@
-/* eslint-disable react/prop-types */
 /* Copyright (c) 2021 Red Hat, Inc. */
 /* Copyright Contributors to the Open Cluster Management project */
 
 'use strict'
 
 import React from 'react'
 import { useQuery } from '@apollo/client'
-import { AcmPageContent, AcmPageHeader, AcmAutoRefreshSelect, AcmRefreshTime, AcmSecondaryNav, AcmAlert, AcmPage } from '@open-cluster-management/ui-components'
+import {
+  AcmPageContent, AcmPageHeader, AcmAutoRefreshSelect,
+  AcmRefreshTime, AcmSecondaryNav, AcmAlert, AcmPage
+} from '@open-cluster-management/ui-components'
 import { Spinner, PageSection } from '@patternfly/react-core'
 import { useHistory, useParams } from 'react-router-dom'
 
-import { INITIAL_POLL_INTERVAL, REFRESH_INTERVALS, REFRESH_INTERVAL_COOKIE } from '../utils/constants'
+import {
+  INITIAL_POLL_INTERVAL,
+  REFRESH_INTERVALS,
+  REFRESH_INTERVAL_COOKIE
+} from '../utils/constants'
 import { getPageDefinition } from './definitions/PageDefinitions'
+import _ from 'lodash'
 
 let timestamp = new Date().toString()
 
+// merge policyAutomations info into policies info
+// by matching policyName + policyNS
+function addAutomationToPolicy(policies, policyAutomations) {
+  const policiesCopy = _.cloneDeep(policies)
+  const policyAutomationsMap = new Map()
+  if (Array.isArray(policyAutomations) && policyAutomations.length > 0) {
+    policyAutomations.forEach((policyAutomation) => {
+      const policyAutomationNS = _.get(policyAutomation, 'metadata.namespace')
+      const policyName = _.get(policyAutomation, 'spec.policyRef')
+      if (policyAutomationNS && policyName) {
+        const policyAutomationsMapKey = `${policyAutomationNS}/${policyName}`
+        policyAutomationsMap.set(policyAutomationsMapKey, policyAutomation)
+      }
+    })
+  }
+
+  if (policyAutomationsMap.size > 0){
+    policiesCopy.forEach((policy) => {
+      const policyNS = _.get(policy, 'metadata.namespace')
+      const policyName = _.get(policy, 'metadata.name')
+      if (policyName && policyNS) {
+        const policyMapKey = `${policyNS}/${policyName}`
+        policy.policyAutomation = policyAutomationsMap.get(policyMapKey)
+      }
+    })
+  }
+  return policiesCopy
+}
+
```

Comment:
It looks like this is creating a new `policyAutomations` key in the `items` object. In that case, we should also update the table definition so that the `msgKey` matches the object so that it can propagate the value to `rawData` and then we can add `sortable: true` to that column (though we'll have to verify whether it's sorting properly when it's `undefined`):
https://github.com/open-cluster-management/grc-ui/blob/60880c88efbaa3e9362e9069f783673a74fb2723/src-web/tableDefinitions/grcPoliciesViewDef.js#L98-L103